src/test/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/api/CardApiTest.java

```diff
@@ -157,4 +159,88 @@ void hide_correct_cards() {
         .jsonPath("_embedded.matches[*].rounds[0].cards[0].cardSuit")
         .isEqualTo(CLUB.name());
   }
+
+  @Test
+  void accept_card_playing() {
+
+    HttpHeaders responseHeaders =
+        webTestClient
+            .post()
+            .uri("/players")
+            .body(BodyInserters.fromValue(PLAYER_1))
+            .exchange()
+            .expectStatus()
+            .isCreated()
+            .expectBody()
+            .returnResult()
+            .getResponseHeaders();
+
+    String sessionIdPlayer1 = getSessionIdOf(responseHeaders);
+
+    responseHeaders =
+        webTestClient
+            .post()
+            .uri("/players")
+            .body(BodyInserters.fromValue(PLAYER_2))
+            .exchange()
+            .expectStatus()
+            .isCreated()
+            .expectBody()
+            .returnResult()
+            .getResponseHeaders();
+
+    String sessionIdPlayer2 = getSessionIdOf(responseHeaders);
+
+    Player player1 =
+        playerRepository.findAll().stream()
+            .filter(player -> player.getName().equals(PLAYER_NAME_1))
+            .findFirst()
+            .orElseThrow();
+
+    Player player2 =
+        playerRepository.findAll().stream()
+            .filter(player -> player.getName().equals(PLAYER_NAME_2))
+            .findFirst()
+            .orElseThrow();
+
+    Game game =
+        GameBuilder.builder("game1", gameRepository, participationRepository, playerRepository)
+            .withParticipationWith(player1)
+            .withParticipationWith(player2)
+            .withGameState(GameState.PLAYING)
+            .withMatch()
+            .withMatchState(MatchState.PLAYING)
+            .withHandForPlayer(PLAYER_NAME_1)
+            .withCards(ACE_OF_CLUBS, QUEEN_OF_CLUBS)
+            .finishHand()
+            .withRound()
+            .finishRound()
+            .finishMatch()
+            .build();
+
+    gameRepository.saveAll(List.of(game));
```

Comment:
```suggestion
    game = gameRepository.saveAll(List.of(game)).get(0);
```
If you save the return value of gameRepository.saveAll, your matches and rounds have ids